Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "MIT OR Apache-2.0"
 name = "omnect-device-service"
 readme = "README.md"
 repository = "git@github.com:omnect/omnect-device-service.git"
-version = "0.18.0"
+version = "0.19.0"
```

Comment:
why is it a minor increase and not a patch? the behavior of the service or the repo doesn't change that much with this little change.
is this PR complete? i'm missing the health-check-scripts and so on to move here?

Reply:
it is correct that omnect-device-service itself doesn't change much, on the other hand a new feature is introduced, so increasing the minor version seemed appropriate to me.

concerning health check scripts I understood that we wanted to have them as part of omnect-device-service only wrt meta layer but not as part of the underlying rust repo.
as the implementation is solely in shell scripts at the moment, changes in health check functionality would then always involve a second repository without real reason imho.
I know there is already a part in omnect-device-service repo (adaption of service file), so I can move the rest over, too, if we do want that.

@JanZachmann what do you think?

Reply:
1st: I would also go for 0.19.0

2nd: From maintenance point of view Harry is right - one vs two PR in case of changes. On the other hand we consequently have everything here (except omnect-device-service.tmpfilesd).
To be honest I'm unsure. Having the files in meta-omnect: recipes-omnect/omnect-device-service/health-check would also be fine for me.

@mlilien do you see other arguments for one or the other? If not I would keep as is in this PR

Reply:
then we should move systemd and sudo and policykit to meta-omnect as well or vice versa.

Reply:
while I do see the advantage to have changes in this one repo only, it's cleaner to me nevertheless to have systemd/sudo/polkit.
why: omnect-device-service is about the daemon code, the functionality itself whereas everything else I see as instrumentation/parameterization of this daemon which depends on the context where it is used.
of course, I can imagine of an opposite argumentation, the more so as omnect-device-service is only used in one context, namely omnect OS.
but it still seems more common to me to put this extra stuff into the meta layer (where I was looking for it to modify right from the start)
finally, moving extra stuff over into meta layer causes discussion about omnect-device-service version change needless here, another argument to me to keep that separate.
just my 0,02 €.

Reply:
after discussion we agreed to move health check stuff to omnect-device-service for the time being.